src/main/Mainpage.hpp

```diff
@@ -22,14 +22,12 @@
  *
```

Comment:
This file is currently failing the code formatting check on CI :rage4: 
If you have `brew` installed (assuming you are on a Mac) you can install `clang-format` by running `brew install clang-format@11`. Once it's installed you can manually run `./.githooks/check-format` from the root of Clio repo and it should correctly split your long lines to fix the format. Let me know if you can't install `clang-format` and I will help with sorting this out for you.

Reply:
Sorry, i think we need to install llvm-17 for this to work.. i suggest you leave this as is and i'll fix later. Let me know when you want me to do it

Reply:
Feel free to fix this now, I've pushed all the other review changes. Appreciate it! 🙏🏾

Reply:
@maria-robobug please apply this diff to fix the formatting to fix the issue:
```diff
diff --git a/src/main/Mainpage.hpp b/src/main/Mainpage.hpp
index 4aef3fc0..079d82b3 100644
--- a/src/main/Mainpage.hpp
+++ b/src/main/Mainpage.hpp
@@ -24,10 +24,13 @@
  *
  * Clio is an XRP Ledger API server optimized for RPC calls over WebSocket or JSON-RPC.
  *
- * It stores validated historical ledger and transaction data in a more space efficient format, and uses up to 4 times less space than <A HREF="https://github.com/XRPLF/rippled">rippled</A>.
+ * It stores validated historical ledger and transaction data in a more space efficient format, and uses up to 4 times
+ * less space than <A HREF="https://github.com/XRPLF/rippled">rippled</A>.
  *
- * Clio can be configured to store data in  <A HREF="https://cassandra.apache.org/_/index.html">Apache Cassandra</A> or <A HREF="https://www.scylladb.com/">ScyllaDB</A>, enabling scalable read throughput.
- * Multiple Clio nodes can share access to the same dataset, which allows for a highly available cluster of Clio nodes without the need for redundant data storage or computation.
+ * Clio can be configured to store data in  <A HREF="https://cassandra.apache.org/_/index.html">Apache Cassandra</A> or
+ * <A HREF="https://www.scylladb.com/">ScyllaDB</A>, enabling scalable read throughput. Multiple Clio nodes can share
+ * access to the same dataset, which allows for a highly available cluster of Clio nodes without the need for redundant
+ * data storage or computation.
  *
  * @section Develop
  *
```
  To apply please save it to a file `diff` in the root of repository and then do `git apply ./diff` and then remove the `diff` file and commit changes.

Reply:
Woot, never knew you could do that - worked perfectly, thanks 😄